lib/apia/open_api/objects/schema.rb

```diff
@@ -80,8 +80,10 @@ def generate_child_schemas
             @children = @definition.fields.values
           elsif @definition.type.argument_set?
             @children = @definition.type.klass.definition.arguments.values
-            @schema[:description] ||=
-              "All '#{@definition.name}[]' params are mutually exclusive, only one can be provided."
+            if @definition.type.klass.ancestors.include?(Apia::LookupArgumentSet)
+              @schema[:description] ||=
+                "All '#{@definition.name}[]' params are mutually exclusive, only one can be provided."
+            end
```

Comment:
change looks good, but can we update the specs to ensure we don't accidentally break this in future?

Reply:
Added a post for the `plain_text` endpoint which uses the regular `KeyValue` argument set in the body. This doesn't use the mutually exclusive description. 
Does that cover what you're looking for?

Reply:
Oh I also disabled the `BlockLength` cop in `example/base.rb` because refactoring that feels like it would make things less readable